sklearn/decomposition/_base.py

```diff
@@ -134,6 +136,8 @@ def inverse_transform(self, X):
         Returns
```

Comment:
In line 132 could you change with

```python
X : array-like of shape (n_samples, n_components)
```

and in line 128, could you put some backticks around `X_original`

Reply:
Thanks, we have already changed it!